### ISSUE TITLE
Smarty with PHP 7.2

### ISSUE THREAD

**Issue by orovellotti** [PnEcrins/FollowDem#55]
We have a warning on one the fonction of Smarty with PHP 7.2 ?

**Comment by camillemonchicourt** [comment]
I dont know...

**Comment by orovellotti** [comment]
"Smarty 3 is PHP 5 only. It will not work with PHP 4."
https://github.com/smarty-php/smarty/blob/master/SMARTY_2_BC_NOTES.txt

**Comment by camillemonchicourt** [comment]
Et ? On bascule sur PHP7, non ?

**Comment by et-taousy** [comment]
We are already on PHP 7.2 just a warning in one Smarty function(deprecated) i will check it.

**Comment by camillemonchicourt** [comment]
Yes but it would be important to update Smarty to a more recent version.
